packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -98,6 +104,25 @@ async fn create_data_type<P: GraphPool>(
     Ok(Json(schema))
 }
 
+#[utoipa::path(
+    get,
+    path = "/data-types",
+    tag = "DataType",
+    responses(
+        (status = 200, content_type = "application/json", description = "Data type found", body = [DataType]),
```

Comment:
Can we update the descriptions in the way described in [this thread](https://github.com/hashintel/hash/pull/885#discussion_r934617983)